sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/implementation/models/LengthUnit.java

```diff
@@ -1,47 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
it is this ok to delete? just wondering.

Reply:
Yeah, since the name and properties are same, moved it over to `models` from `implementation.models`